front/src/modules/trainschedule/components/DriverTrainScheduleV2/DriverTrainScheduleStopListV2.tsx

```diff
@@ -35,9 +35,9 @@ const DriverTrainScheduleStopListV2 = ({
 
   const formattedMrsp = useMemo(
     () =>
-      mrsp.positions.map((_position, index) => ({
+      mrsp.boundaries.map((_position, index) => ({
```

Comment:
I think it would be better to map on the values rather than on the boundaries, otherwise you might miss the last value.
Moreover, I think you need to -1 to get the correct boundary
```suggestion
      mrsp.values.map(({ speed }, index) => ({
        position: index === 0 ? 0 :  mrsp.boundaries[index-1],
        speed
      })
```

Reply:
The handling of the response was indeed wrong.
Yet, the path boundaries are not useful, so I chose to work with the actual MRSP in https://github.com/OpenRailAssociation/osrd/pull/8351/commits/b4c6f6f013b673a29e906c167d7bc2d24f5f92a0#diff-e7728b2c01bcf96a9b19c23cff25682e233aea4a7dcee40f4615cb88e38d28e9R38-R39 (same for CSV to mutualize)